mindsdb/interfaces/native/mindsdb.py

```diff
@@ -103,9 +103,8 @@ def rename_model(self, name, new_name):
         self.dbw.register_predictors(self.get_model_data(new_name), setup=False)
 
     def load_model(self, fpath):
-        F.import_model(model_archive_path=fpath)
-        # @TODO How do we figure out the name here ?
-        # dbw.register_predictors(...)
+        name = F.import_model(model_archive_path=fpath)
+        self.dbw.register_predictors(self.get_model_data(new_name), setup=False)
```

Comment:
`name` instead `new_name`?

Reply:
Error indeed, we should add test to cover it...